Modix/ModixBot.cs

```diff
@@ -124,13 +124,13 @@ public async Task HandleCommand(SocketMessage messageParam)
             {
                 var context = new CommandContext(_client, message);
 
+                if (!(context.User is IGuildUser)){ return; }
```

Comment:
We should plan to support non-guild users eventually. There are some commands, like help, that we can serve up in DM channels. I forget if it was rowboat or MODiX or another bot that already does this.

Reply:
Agreed. I put it in for now as many parts of Modix expect a guild context, if I recall.